### PR TITLE
Add some more details about gas costs

### PR DESCRIPTION
**Pull request by peroket**

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -120,6 +120,28 @@ const AboutPage = () => {
         calling contract with <OLink opcode="3D" title="RETURNDATASIZE" /> and{' '}
         <OLink opcode="3E" title="RETURNDATACOPY" />.
       </p>
+
+      <H2 className="mb-6">Gas costs</H2>
+      <p className="pb-8">
+        As an incentive to provide resources to run transactions, a fee is payed
```
Comment (assimovt) [inline pages/about.tsx:126]: `paid` for financial meaning

File: pages/about.tsx [hunk 2]
```diff
@@ -120,6 +120,28 @@ const AboutPage = () => {
...
+        to send and execute a transaction. The fee is determined by several
+        factors, including the amount of data sent or the amount of work that a
+        transaction requires. That fee is calculated through two mechanisms. The
+        first is a fixed cost that is defined by Ethereum, depending on what is
+        done. The unit for that is called gas, and reùains the same, though it
```
Comment (assimovt) [inline pages/about.tsx:131]: maybe clarify `done`? Is it executed? or what done means?

Comment (assimovt) [reply pages/about.tsx:131]: fix `reùains`

File: pages/about.tsx [hunk 3]
```diff
@@ -120,6 +120,28 @@ const AboutPage = () => {
...
+        can be changed by a hardfork. The second component is the price of one
+        gas unit, which varies over time according to what people are willing to
+        pay to run their transactions. Its unit is ethereum per gas.
```
Comment (assimovt) [inline pages/about.tsx:134]: should it be `ETH` per gas?

File: pages/about.tsx [hunk 4]
```diff
@@ -120,6 +120,28 @@ const AboutPage = () => {
...
+      </p>
+
+      <p className="pb-8">
+        Each opcode has its own gas cost. Their are two parts to the gas cost.
```
Comment (assimovt) [inline pages/about.tsx:138]: `their` => `there`

**Review by assimovt — CHANGES_REQUESTED** [review]
Some minor typos

File: pages/about.tsx (hunk 1 above)
Comment (peroket) [reply pages/about.tsx:126]: done

File: pages/about.tsx (hunk 3 above)
Comment (peroket) [reply pages/about.tsx:134]: done

File: pages/about.tsx (hunk 2 above)
Comment (peroket) [reply pages/about.tsx:131]: "executed" is better thanks, done

File: pages/about.tsx (hunk 4 above)
Comment (peroket) [reply pages/about.tsx:138]: oups thanks fixed

**Review by assimovt — APPROVED** [review]
🍰

**Pull request merged**